### ISSUE TITLE
Cannot reference Serializer before use 

### ISSUE THREAD

**Issue by steel971** [silverlogic/djangorestframework-expander#4]
Hi,
I discover a little bug i cannot resolve with the actual Mixin, here the code :

        class KidSerializer(ExpanderSerializerMixin, serializers.ModelSerializer):
                guardian_kid = serializers.HyperlinkedRelatedField(view_name='guardian-kid-detail')

                class Meta:
                    model = GuardianKid
                    fields = (
                        'guardian_kids',
                    )
                    expandable_fields = {
                        'guardian_kids': (GuardianSerializer, (), {'many'=True}
                    }

        class GuardianKidSerializer(ExpanderSerializerMixin, serializers.ModelSerializer):
                kid = serializers.HyperlinkedRelatedField(view_name='kid-detail')

                class Meta:
                    model = GuardianKid
                    fields = (
                        'kid',
                    )
                    expandable_fields = {
                        'kid': KidSerializer
                    }

With that, you cannot use **GuardianSerializer** before you declare it (that's normal). So I tried to handle this like in [Django Model Foreign Key](https://docs.djangoproject.com/en/1.8/ref/models/fields/#foreignkey) by using the name instead of the class

        class KidSerializer(ExpanderSerializerMixin, serializers.ModelSerializer):
                (...)
                expandable_fields = {
                    'guardian_kids': ('GuardianSerializer', (), {'many'=True}
                }

But it raise an error : _the first parameters has to be a class_. I also try to hook the __init__ in order to declare an self.Meta.expandable_fields

I think that can be handle easily like in Django, in file _expander/serializers.py_ at the line 41 instead of just look for a Class, you can try if it's a Class or a String.

    # Using sys.module <<< IMHO I'll go with this one
    import sys
    getattr(sys.modules[__name__], String)

    # Using eval !!!WARNING!!! is commonly admit eval() is a bad practice
    eval(String)

    # Using global()
    globals()[String]

It's not an EndUser input, so it's not so dangerous, but i let you the choice on the method you prefer.
I hope that will help !

**Comment by ryanpineo** [comment]
Good point and good suggestions, I'll take a look at the django FK implementation and see how they do it.  If you want to check out the django FK yourself and make a PR along the same lines it would be appreciated.

**Comment by steel971** [comment]
I forgot to mention it, but I already looked at the Django FK implementation, but I don't think this implementation is usable here :

    # django/django/db/models/fields/related.py :  l.795
    def __init__(self, to, on_delete=None, related_name=None, related_query_name=None,
                 limit_choices_to=None, parent_link=False, to_field=None,
                 db_constraint=True, **kwargs):
        try:
            to._meta.model_name
        except AttributeError:
            assert isinstance(to, six.string_types), (
                "%s(%r) is invalid. First parameter to ForeignKey must be "
                "either a model, a model name, or the string %r" % (
                    self.__class__.__name__, to,
                    RECURSIVE_RELATIONSHIP_CONSTANT,
                )
            )
        else:
            # For backwards compatibility purposes, we need to *try* and set
            # the to_field during FK construction. It won't be guaranteed to
            # be correct until contribute_to_class is called. Refs #12190.
            to_field = to_field or (to._meta.pk and to._meta.pk.name)

[The **_meta method** can be use](https://docs.djangoproject.com/en/1.10/ref/models/meta/#module-django.db.models.options) only with Django model :-(
